src/commands/manifest/import.ts

```diff
@@ -7,18 +7,26 @@ import * as importService from 'services/import-manifest-service';
 import { PromptService } from 'services/prompt-service';
 import { ImportCommandTasksContext } from 'types/commands/manifest-import';
 import logger from 'utils/logger';
+import { parseTemplateVars } from 'utils/template-vars';
 
 const MESSAGES = {
   path: 'Path to your app manifest file on your machine',
   appId: 'App id (will create a new draft version)',
   appVersionId: 'App version id to override',
   newApp: 'Create new app',
+  templateVars: 'Template variables in format KEY=VALUE,KEY2=VALUE2',
 };
 
 export default class ManifestImport extends AuthenticatedCommand {
-  static description = 'Import manifest.';
+  static description = 'Import manifest with optional template variables.';
   static withPrintCommand = false;
-  static examples = ['<%= config.bin %> <%= command.id %>'];
+  static examples = [
+    '<%= config.bin %> <%= command.id %>',
+    '<%= config.bin %> <%= command.id %> -p ./manifest.json',
+    '<%= config.bin %> <%= command.id %> --manifestPath ./manifest.json --templateVars "THANKS_GOD=thanks_goda"',
```

Comment:
Better to load it from the env as these commands may be printed in logs and expose secret values